fedimint-api/src/db/mod.rs

```diff
@@ -93,8 +93,8 @@ dyn_newtype_define! {
 /// | Type     | Non-Readable Write | Dirty Read | Non-Repeatable Read | Phantom Record | Lost Writes |
```

Comment:
Before merging this, I want to move DatabaseOperation and DatabaseInsertOperation/DatabaseDeleteOperation into mem_impl.rs since that is the only place it is used.